packages/web_benchmarks/lib/src/browser.dart

```diff
@@ -237,14 +238,15 @@ String _findSystemChromeExecutable() {
   } else if (io.Platform.isWindows) {
     const String kWindowsExecutable = r'Google\Chrome\Application\chrome.exe';
     final List<String> kWindowsPrefixes = <String>[
-      io.Platform.environment['LOCALAPPDATA'],
-      io.Platform.environment['PROGRAMFILES'],
-      io.Platform.environment['PROGRAMFILES(X86)'],
+      for (String? item in <String?>[
+        io.Platform.environment['LOCALAPPDATA'],
+        io.Platform.environment['PROGRAMFILES'],
+        io.Platform.environment['PROGRAMFILES(X86)'],
+      ])
+        if (item != null) item
```

Comment:
You can also do `...array.whereNotNull()`

https://api.flutter.dev/flutter/package-collection_collection/IterableNullableExtension/whereNotNull.html

Reply:
this got off my radar, will update this today.

Reply:
```
...<String?>[
  io.Platform.environment['LOCALAPPDATA'],
  io.Platform.environment['PROGRAMFILES'],
  io.Platform.environment['PROGRAMFILES(X86)'],
].whereType<String>()
```

could we do this ?